app/views/projects/edit.html.erb

```diff
@@ -1,7 +1,13 @@
 <% content_for :title_bar do %>
   <ul class="nav navbar-nav">
     <li>
-      <%= link_to @project.name, @project %>
+      <%= link_to @project.name, @project, { class: "project-tag" } %>
+      <% unless @project.tag_group.blank? %>
```

Comment:
You're using this code into other files, It's a better idea to create a partial for it.